ui/panels-plugin/src/plugins/bar-chart/BarChartOptionsEditorSettings.tsx

```diff
@@ -65,6 +66,17 @@ export function BarChartOptionsEditorSettings(props: BarChartOptionsEditorProps)
     );
   };
 
+  const handleResetSettings: React.MouseEventHandler<HTMLButtonElement> = () => {
+    onChange(
+      produce(value, (draft: BarChartOptions) => {
+        draft.calculation = 'LastNumber';
+        draft.unit = DEFAULT_UNIT;
+        draft.sort = 'desc';
+        draft.mode = 'value';
```

Comment:
Definitely up to you, but we could potentially make these all constants and use them as the fallback value in the inputs below too (i.e. use them both in this reset function and as the defaults for each input) 

It's more verbose, so up to you if you think it's more helpful

Reply:
Good call, the defaults were being defined in a few places so I think it's good to set them as constants. Updated! [6d798df](https://github.com/perses/perses/pull/1325/commits/6d798dfe9e04be2956d98d45cbc7dba464cb14e8)